src/FunctionApps/python/IntakePipeline/transform.py

```diff
@@ -1,2 +1,68 @@
-def transform_bundle(bundle: dict) -> None:
-    pass
+from smartystreets_python_sdk import us_street
+
+from phdi_transforms.basic import transform_name, transform_phone
+from phdi_transforms.geo import geocode
+
+
+def find_patient_resource(bundle: dict) -> dict:
+    """Grab a patient resource out of the bundle, and return a reference"""
+
+    patients = [
+        r
+        for r in bundle.get("entry")
+        if r.get("resource").get("resourceType") == "Patient"
+    ]
+
+    if patients:
+        return patients[0]
```

Comment:
This won't work if there are multiple patient resources in a bundle. In theory I don't think that should happen, but it is something to be aware of.

Reply:
I would prefer to see us code things to handle all patients in a bundle. While I do agree that theoretically that shouldn't happen, it's a small defensive thing we can do to make our code more robust and handle something I don't think we can/should safely assume.